dev/io.openliberty.springboot.support.shutdown.version40/bnd/springBoot40Closer.bnd

```diff
@@ -1,4 +1,3 @@
 Bundle-SymbolicName: springBoot40Closer
 Private-Package: io.openliberty.springboot.support.shutdown.*
-Include-Resource: resources/container
```

Comment:
This project should no longer need to build this inner JAR `springBoot40Closer.jar` since the classes get pulled into the other project's JAR now.  At some point we should remove this and remove the `-make= (*).(jar); type=bnd; recipe="bnd/$1.bnd"` instruction from this project's `bnd.bnd`

Reply:
Right! Thanks, Tom!